test/gurka/gurka.h

```diff
@@ -582,6 +582,11 @@ route(const map& map, const std::string& from, const std::string& to, const std:
   return route(map, {from, to}, costing);
 }
 
+valhalla::Api route(const map& map, const std::string& request_json) {
```

Comment:
cc @danpat I added this for convenience of writing tests with additional parameters in the request. If you have a neater pattern in mind for custom request parameters let me know.